src/tools/material.js

```diff
@@ -18,6 +21,27 @@ function scaleOneLatticeVector(material, key = 'a', factor = 1.0) {
 
 }
 
+/**
+ * Returns the lattice for a non-periodic structure. The lattice has been scaled to accomodate the size of the
+ * non-periodic structure.
+ * @param material {Material}
+ * @returns {Object}
+ */
+function scaleLatticeToMakeNonPeriodic(material) {
+    const basis = new Basis({
+        ...material.Basis.toJSON(),
+    })
+    basis.toCartesian();
+    const scalingFactor = nonPeriodicLatticeScalingFactor;
+    const newLattice = new Lattice({
+        a: basis.maxPairwiseDistance * scalingFactor,
+        type: "CUB"
+    });
+
+    return newLattice.toJSON();
+}
```

Comment:
(1) I'd just implement the function to be modifying material in-place like the `scaleOneLatticeVector` above.
(2) why do we need to convert basis to cartesian? the `maxPairwiseDistance` should be taking care of that already on its own. I think the distance only makes sense in cartesian coordinates.
(3) the implementation should just be

```js
function scaleLatticeToMakeNonPeriodic(material) {
    material.lattice = new Lattice({
        a: material.Basis.maxPairwiseDistance * nonPeriodicLatticeScalingFactor,
        type: "CUB"
    });
}
```